site-feature.js

```diff
@@ -168,9 +168,7 @@ function f2html(fdata) {
     ihtml = ihtml + '<b>' + l10n.email + ': </b>' + genmailto(fdata.properties['email']) + '<br />';
   }
 
-  if ("phone" in fdata.properties) {
-    ihtml = ihtml + '<b>' + l10n.phone + ': </b>' + fdata.properties['phone'] + '<br />';
-  }
+  ihtml += phoneNumberHTML(fdata);
```

Comment:
I think the `if` statement needs to stay in someway, otherwise there is an `undefined` display if a campsite does not have a phone number tagged.

Compare https://camping.openstreetmap.de/map-dev/#14/48.0838/9.1380/0/1/bef/way/885634788 to other sites around with a phone number tagged...

![grafik](https://user-images.githubusercontent.com/3876790/182824382-92b93614-1529-4405-8971-436ad26013e1.png)

Reply:
Yes, good point, thanks for raising this! The `if` statement has moved to the implementation of `phoneNumberHTML()`, as an early `return` statement. That way, the calling code (the place that you commented) becomes less cluttered with `if` statements, whereas the actual logic has the responsibility of checking for the existence of the `phone` tag.